src/index.ts

```diff
@@ -1,3 +1,4 @@
 export { CALLDATA as Calldata } from './calldata/index';
 export { Client } from './client';
 export { EXTERNAL as Constants } from './constants';
+export * from './util'
```

Comment:
I'd like to mimic what we are doing with `Constants` in the line above. I find it cleaner to have all utils organized in one export. Users can always reconstruct.

Reply:
[bd00fbc](https://github.com/GridPlus/gridplus-sdk/pull/371/commits/bd00fbcd9bf6a01bd793069953741b78247ba32e)